### PR TITLE
preparing the app for first release

### PR DESCRIPTION
**Pull request by helloseyedjafari**

### CODE REVIEW

File: navigation/src/androidMain/kotlin/com/islamversity/navigation/changehandler/ArcFadeMoveChangeHandler.kt [hunk 1]
```diff
@@ -2,13 +2,11 @@ package com.islamversity.navigation.changehandler
...
 import android.view.View
 import android.view.ViewGroup
 import androidx.annotation.Keep
-import com.bluelinelabs.conductor.changehandler.SharedElementTransitionChangeHandler
-import com.bluelinelabs.conductor.internal.LegacyTransitionUtils
+import androidx.transition.Transition
+import com.bluelinelabs.conductor.changehandler.androidxtransition.SharedElementTransitionChangeHandler
 import java.util.*
 
 @TargetApi(Build.VERSION_CODES.LOLLIPOP)
```
Comment (rezaiyan) [inline navigation/src/androidMain/kotlin/com/islamversity/navigation/changehandler/ArcFadeMoveChangeHandler.kt]: This class is useless!

File: navigation/src/androidMain/kotlin/com/islamversity/navigation/changehandler/ArcFadeMoveChangeHandlerCompat.kt [hunk 2]
```diff
@@ -1,16 +1,34 @@
...
     @Keep
     constructor() : super() {
     }
 
-    constructor(vararg transitionNames: String) : super(
-        ArcFadeMoveChangeHandler(*transitionNames),
-        FadeChangeHandler()
-    )
+    override fun configureSharedElements(container: ViewGroup, from: View?, to: View?, isPush: Boolean) {
+        TODO("Not yet implemented")
```
Comment (rezaiyan) [inline navigation/src/androidMain/kotlin/com/islamversity/navigation/changehandler/ArcFadeMoveChangeHandlerCompat.kt]: these TODOs throw `NotImplementedError`

**Review by rezaiyan — APPROVED** [review]
Thank you

File: navigation/src/androidMain/kotlin/com/islamversity/navigation/changehandler/ArcFadeMoveChangeHandler.kt (hunk 1 above)
Comment (helloseyedjafari) [reply navigation/src/androidMain/kotlin/com/islamversity/navigation/changehandler/ArcFadeMoveChangeHandler.kt]: removed

File: navigation/src/androidMain/kotlin/com/islamversity/navigation/changehandler/ArcFadeMoveChangeHandlerCompat.kt (hunk 2 above)
Comment (helloseyedjafari) [reply navigation/src/androidMain/kotlin/com/islamversity/navigation/changehandler/ArcFadeMoveChangeHandlerCompat.kt]: was not calling them
anyways
removed

**Pull request merged**